### ISSUE TITLE
DOC: Add instructions for how to get started with auth & GCP

### ISSUE THREAD

**Issue by tswast** [pydata/pydata-google-auth#19]
Many of the instructions at https://pandas-gbq.readthedocs.io/en/latest/howto/authentication.html also apply to this library and whereever else it is used, such as in Ibis.

We should pull relevant how-tos out of pandas-gbq and into the documentation for pydata-google-auth.

**Comment by max-sixty** [comment]
👍 

Somewhat separately: IMHO those present a fairly complicated view of what people need to do to get up & running. We've a (so far) v successful setup here where everyone runs `gcloud auth login` and `gcloud auth application-default login`, and that is all anyone needs to do, ever. The same code runs in GKE/GCE. It's also easier and more secure than AWS!

Some of those steps look useful for _library_ developers, but to run some data science it looks a lot, particularly given the audience is generally less technical. It's certainly possible I'm misunderstanding something so please correct me if I'm wrong.

In summary: I'd have a banner at the top saying "If you're just getting started, run `gcloud auth application-default login`, and go do some awesome data work. See below if you need more customizations"

**Comment by tswast** [comment]
> In summary: I'd have a banner at the top saying "If you're just getting started, run gcloud auth application-default login, and go do some awesome data work. See below if you need more customizations"

The only caveat with that is there are still rumblings within Google such as the thread at https://github.com/googleapis/google-auth-library-python/issues/271 about turning that feature off. :-( Thus, `pydata_google_auth.default()` is the next best thing, since it'll fall back to the web browser auth flow.

**Comment by max-sixty** [comment]
Yeah. Maybe let's see how that evolves. 

From a user perspective: I think that would create non-trivial friction to getting up & running, exactly for these sorts of use cases. But I only have a partial view
